test/functional/test.run.js

```diff
@@ -124,20 +124,9 @@ describe("jpm run", function () {
       var options = { cwd: paramDumpPath, env: { JPM_FIREFOX_BINARY: binary }};
       var proc = exec("run", options, function (err, stdout, stderr) {
         expect(err).to.not.be.ok;
-        expect(stdout).to.contain("console\.log: param-dump:")
+        expect(stdout).to.contain("console\.log: param-dump:");
         expect(stdout).to.contain("PARAMS DUMP START");
-        done();
-      });
-    });
-
-    it("logs out everything from console with -v", function (done) {
```

Comment:
Well the problem is that we strip logs in testing, as I mentioned. So we could remove that but then lots of other test will probably fail. :confused:

This test working is really just an accident because we forgot to log `Creating XPI` correctly. It didn't even check what it was supposed to check.

In the end it's your call, but I would suggest moving the "not stripping logs in tests anymore" into a separate pull request, which would mean no tests for this.